### DISCUSSION TITLE
Seeking Community Feedback on Motion Capture Calibration Process

### DISCUSSION THREAD

**Discussion by lbaron42** [jyjblrd/Mocap-Drones#68] [Q&A]
Hello everyone!

We are students of [42 Berlin—Software Training Institute](https://42berlin.de/de/), and we are engaged in an art/educational project involving dancing drones interacting with dancers on a stage. After long research, we found the "Low-Cost-Mocap" video on YouTube and adopted it as the basis of our project within our limited time frame.

After two months, we have yet not passed the calibration process, and we have decided to call for help from the community.

We created a video based on what we understand about the calibration process, trying to explain it:

[Watch our calibration video](https://youtu.be/AFp89tyswrY?si=kWq9LPH4fonFxNxs)

[![Watch our calibration video](https://img.youtube.com/vi/AFp89tyswrY/maxresdefault.jpg)](https://youtu.be/AFp89tyswrY?si=kWq9LPH4fonFxNxs)

We also want to create a community around this project. We improved the camera calibration process and are willing to create more solid and simple documentation for it as soon as we get it to work.

Anyone willing to share some knowledge or even engage in a video conference to exchange experiences, it will be a pleasure to chat.

Just DM me on:

- [WhatsApp](https://wa.me/5541991684010)
- [Telegram](https://t.me/lbaron1984)
- [LinkedIn](https://www.linkedin.com/in/luiz-gabriel-baron-98999b23b)
- Email: [lbaron@student.42berlin.de](mailto:lbaron@student.42berlin.de)

**Comment by gumby0q** [comment]
Hi, @jyjblrd and community
can you give any hints, references for good some books or articles about that part of computer vision and bundle adjustment process?
so far I have found some example project that use a bit different modified approach from that repository (different 'markers', and with a bit different optimization params).   
https://github.com/mprib/caliscope
and referenced there scipy example
https://scipy-cookbook.readthedocs.io/items/bundle_adjustment.html

so to be more specific I'm personally struggle with:
1. Is there any really good book that can enlighten things about cv and math used in that project? I have lack of knowledge in that area and really go from bottom of understanding.
2. Any hints how to validate camera pose calibration numerically or with some devices, special tools or something? And especially if there is needed really good results to get tracking working even if the some cameras are not getting to the markers.
3. Any directions to methods, articles or ideas how to improve tracking and calibration?

Sincerely yours,
gumby0q

**Reply by yangjun1222** [reply]
I am reading this project file. Understanding the algorithms is important. Following link maybe helpful.
https://www.mathworks.com/matlabcentral/fileexchange/67383-stereo-triangulation
https://www.mathworks.com/matlabcentral/fileexchange/78955-multi-view-triangulation
https://www.mathworks.com/help/vision/camera-calibration.html
